gradle.properties

```diff
@@ -45,7 +45,7 @@ buildFromSource=true
 
 # The default version for LabKey artifacts that are built or that we depend on.
 # override in an individual module's gradle.properties file as necessary 
-labkeyVersion=23.2-SNAPSHOT
+labkeyVersion=1.39.0-refactor-SNAPSHOT
```

Comment:
@labkey-tchad this doesn't look right...

Reply:
whoops